CHANGES.rst

```diff
@@ -6,6 +6,8 @@ Version 0.5.4
 
 To be released.
 
+- Fixed `nirum.datastructures.List` that shouldn't refer defined value.
```

Comment:
이쯤으로 바꾸면 어떨까요?

```
- Made `nirum.datastructures.List` to copy the given value so that
  it doesn't refer given value's state and is immutable.
```